admin/worker_v2.m.template

```diff
@@ -13,9 +13,62 @@ function [ok,err_mess,je]=worker_v2(worker_controls_string)
 %              contains only minimal initialization information, namely the
 %              folder name where the job initialization data are located on
 %              a remote system.
-
+je = [];
 if isempty(which('herbert_init.m'))
-    horace_on();
+    try
+        horace_on()
+    catch ME
+        ok = false;
+        err_mess = ME;
+        pid = int64(feature('getpid'));
+        log_file = sprintf('WORKER_V2_pid%i_PackageInit_failure.log',pid);
+        write_fail_log(log_file,err_mess)
+        return;
+    end
 end
 DO_LOGGING = false;
-[ok, err_mess,je] = parallel_worker(worker_controls_string,DO_LOGGING);
+DO_DEBUGGING = false;
+try
+    [ok, err_mess,je] = parallel_worker(worker_controls_string,DO_LOGGING,DO_DEBUGGING);
+catch ME1 % intercepted exception in processing failure or some odd bug indeed
+    pid = int64(feature('getpid'));
+    ok = false;
+    if exist('je','var') && isa(je,'JobExecutor') %exception thrown but JE is defined
+        if isnumeric(je.labIndex) && ~isempty(je.labIndex)
+            log_file = sprintf('WORKER_V2_pid%i_Node#%d_failure.log',pid,je.labIndex);
```

Comment:
filename pattern should be a constant (ideally shared with _4tests.template)

Reply:
as no logs are printed anyway, probably no point in complicating the logic